ehitchhiking-ui/src/app/shared/components/trips-modal/trips-modal.component.sass

```diff
@@ -0,0 +1,56 @@
+@import '../../../../styles/fonts-constants'
+@import '../../../../styles/colors'
+@import '../../../../styles/mixins'
+@import '../../../../styles/modals'
+
+.form
+  height: 100%
+  display: flex
+.center
+  margin: auto
+  text-align: center
+.trips-modal-container
+  position: relative
+.trips-title
+  text-align: center
+  margin-bottom: 0
+  font-size: $title
+.form-row
+  display: flex
+  justify-content: center
+  align-items: center
+  .icon-size
+    width: 5rem
+.scroll-marker
+  min-height: 20px
+.trips
+  height: calc(22.5rem + 10px)
```

Comment:
pls add comment what is 22.5rem used for. the same for 10px

Reply:
22.5 = 4.5*5, 4.5 - one trip,
10 px - height empty div; it's for scrolling